packaging/windows/msi/mk.debug

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env bash
```

Comment:
When is this script executed? It seems like the Makefile only ever executes `mk_msi.$(1)`. Can this file be removed?

Reply:
it is used by developer. for local verification. old signing process.

Reply:
I guess my question stands, is there any utility in keeping this script around still?

Reply:
Yes. It  is used to sign debug build locally. I use it.